app/components/global/AppBar/AppBar.js

```diff
@@ -77,11 +77,6 @@ AppBar.propTypes = {
       link: PropTypes.string.isRequired,
     }),
   ).isRequired,
-  user: PropTypes.shape({}),
```

Comment:
`user` is still a prop that's getting passed into this component, so I wouldn't suggest removing PropTypes altogether. Just clarifying the type?

Reply:
We could always raise a new issue for this, rather than fixing in this PR?

Reply:
I'll leave this out of this PR for now. I'm unsure what the correct format for this should be as the linter blocks the generic `object` type and this component doesn't really need to know anything about the `user` as it just forwards it on to the `ProfileMenu`. I was reluctant to remove this as I've always thought that if using propTypes in a component then all props should be assigned but it also felt redundant adding the code for the user shape to a component that doesn't care about its format.

Reply:
Yeah I totally get what you mean. But I reckon, if we're going to use PropTypes, let's use it consistently everywhere. I think it's really good that you've been adding it to files that are missing them. As for this, specifically, I reckon it's worth duplicating the specific shape of the user object, both here in this file & in the ProfileMenu file.

When we start seeing enough areas of duplication, that might be time to refactor out into a generic types file, to be imported as and when needed. I'm not sure that we're there yet. But ensuring that we have PropTypes on all files will be a good way to see when we've got that level of duplication.

And yeah, sorry, I didn't realise the linter blocks you from just saying `object`. But I think that's an attempt to get you to specify the _shape_ of the object, perhaps?

So just like [line 74](https://github.com/elifesciences/elife-xpub/pull/927/files/81a6776b624e4eb17cb2927d4012ad6187323bdd..ff9a8a42a972891b810d0c5398b5a161403f8b90#diff-db536dc0676835888e436d994badfc32R74), you would do
```js
PropTypes.objectOf(
  ProTypes.shape({
    identities
    .... etc (whatever else needs to be on here - haven't looked)
  })
})
```